### PR TITLE
feat: (TNLT-4211) Supplement Lead One and Four Slice

### PR DESCRIPTION
**Pull request by aronshamash**
- Supplement Lead One and Four Slice for tablets only
- https://nidigitalsolutions.jira.com/browse/TNLT-4211

### iPad (7th gen) (Portrait)
Screenshot 2020-09-02 at 14 45 39

### iPad  (7th gen) (Landscape)
Screenshot 2020-09-02 at 14 46 47

### iPad Mini 4 (Portrait)
Screenshot 2020-09-02 at 14 52 08

### iPad Mini  4 (Landscape)
Screenshot 2020-09-02 at 14 52 22

### iPad Pro 12.9 inch (4th gen) (Portrait)
Screenshot 2020-09-02 at 14 45 33

### iPad Pro 12.9 inch (4th gen) (Landscape)

Screenshot 2020-09-02 at 14 46 42

### Nexus 9 (Android) (Portrait)

Screenshot 2020-09-03 at 10 08 45

### CODE REVIEW

File: packages/edition-slices/src/slices/shared/flush-responsive-slice.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { ResponsiveContext } from "@times-components-native/responsive";
+import { editionBreakpoints } from "@times-components-native/styleguide";
+
+const FlushResponsiveSlice = ({
```
Comment (c-j-j) [inline packages/edition-slices/src/slices/shared/flush-responsive-slice.js:5]: As this and ResponsiveSlice are similar except for the horizontal margin, would it also work if we passed a prop to the original one called `flush` that then removes that margin?

I'm just thinking if we do start supporting more breakpoints, we'll need to update both this and the ResponsiveSlice components.

File: packages/edition-slices/src/tiles/tile-az/styles/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  [editionBreakpoints.medium]: spacing(10),
+  [editionBreakpoints.wide]: spacing(28),
+};
+
+export default (breakpoint) => ({
+  container: {
+    flex: 1,
+    borderBottomWidth: 1,
+    borderBottomColor: colours.functional.keyline,
+    marginHorizontal: -horizontalMarginResolver[breakpoint],
```
Comment (c-j-j) [inline packages/edition-slices/src/tiles/tile-az/styles/index.js:28]: Might be worth checking on an android nexus 9 that this negative margin does the right thing?

**Review by c-j-j — APPROVED** [review]

File: packages/edition-slices/src/slices/shared/flush-responsive-slice.js (hunk 1 above)
Comment (aronshamash) [reply packages/edition-slices/src/slices/shared/flush-responsive-slice.js:5]: I wanted to go with your suggestion approach originally but it got a bit messy.

File: packages/edition-slices/src/tiles/tile-az/styles/index.js (hunk 2 above)
Comment (aronshamash) [reply packages/edition-slices/src/tiles/tile-az/styles/index.js:28]: Nexus 9 all good. I've added to screenshots in desc of PR now.

File: packages/edition-slices/src/edition-slices.js [hunk 3]
```diff
@@ -22,8 +22,9 @@ import {
   LeadOneFullWidthFrontSlice,
   PuzzleSlice,
   TopSecondarySlice,
-  SupplementSecondaryFourSlice,
+  SupplementLeadOneAndFourSlice,
   SupplementLeadOneAndOneSlice,
+  SupplementSecondaryFourSlice,
```
Comment (Geoff-Ford) [inline packages/edition-slices/src/edition-slices.js:27]: Why you be like that?

**Review by Geoff-Ford — APPROVED** [review]

File: packages/edition-slices/src/edition-slices.js (hunk 3 above)
Comment (aronshamash) [reply packages/edition-slices/src/edition-slices.js:27]: alphabetically order but it is not consistent throughout the codebase

**Pull request merged**